### ISSUE TITLE
Form

### ISSUE THREAD

**Issue by LilyLiuweb** [LilyLiuweb/youtube#4]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>驗證表單</title>
    <script type="text/javascript" src="/jquery.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
    <style>

        body{
            background-color: #F2F3F5;
        }

        #container{

            padding-top: 60px;
            width:600px;
            margin: auto;
            font-family:"Helvetica Neue" , "Helvetica" ,"Arial";
        }
        .input-style{
            border-radius: 5px;
            border: 1px solid grey;
            font-size:1.1em;
            padding-left: 20px;
            margin-bottom: 10px;

        }
        .input-width{
            width:300px;
            height: 40px;
        }
        label{
            padding-top: 8px;
            width: 200px;
            float: left;
            font-size:1.2em;
        }
        #mail-gender{
            width:323px;
            height: 40px;
        }
        #email-msg{
            width: 500px;
            padding-top: 15px;
        }
        #submitButton{
            background-color: #AA47BC;
            color: white;
            width: 525px;
            height: 40px;
        }
        #error{
            color: red;
        }
    </style>
</head>
<body>
    
    <div id="container">
        <form id="vaildationForm">
            <label for="email-name">姓 名</label>
            <input id="email-name" class="input-style input-width" type ="text" name="name" placeholder="請輸入您的姓名">

            <label for="email-address">電子郵箱</label>
            <input id="email-address" class="input-style input-width" type ="text" name="name" placeholder="請輸入您的EMAIL">    

            <label for="mail-gender">性 別</label>
            <select id="mail-gender"class="input-style" name="gender">
                <option value="male">男 性</option>
                <option value="female">女 性</option>
            </select>

            <label for="pass1">密 碼</label>
            <input id="pass1" class="input-style input-width" type ="password" name="pass1" placeholder="請輸入您的密碼">

            <label for="pass2">確認密碼</label>
            <input id="pass2" class="input-style input-width" type ="password" name="pass2" placeholder="請再次輸入您的密碼">

            <textarea id="email-msg" class="input-style" name="message" rows="8" cols="80"  placeholder="請描述您的狀況"> </textarea>
            <button id="submitButton" class="input-style" type="submit" name="submit">發送郵件</button>
        </form>

        <div id="error">

        </div>
        <script type="text/javascript">

            $('#vaildationForm').submit(function(event){

                var errorMessage = "";

                event.preventDefault();

                var name = $("#email-name").val();
                var email = $("#email-address").val();
                var pass1 = $("#pass1").val();
                var pass2 = $("#pass2").val();
                var message = $("#email-msg").val();

                function isValidEmailAddress(emailAddress){

                    var pattern = /^\b[A-Z 0-9-]+@[A-Z 0-9]+\.com\b/i;

                    return pattern.test(emailAddress);
                }

                if(name == "" || email == ""|| pass1 == "" || pass2 == "" || message == "" ){
                    
                    errorMessage = "請輸入完整信息.";

                }else if( !isValidEmailAddress(email)){

                    errorMessage = "請輸入正確郵箱地址.";
                }else if(pass1 != pass2){

                    errorMessage = "請輸入正確密碼";
                }

                if (errorMessage == ""){
                    $("#error").html("郵件已經發送");
                    $("#error").css("color","green");

                } else {

                    $("#error").html(errorMessage);
                    $("#error").css("color","red");

                }

                   
            });
        </script>
    </div>
</body>
</html>
